packages/release-mastermind/src/index.ts

```diff
@@ -3,96 +1,160 @@
+/** @format */
+
+import * as core from "@actions/core"
+import {getOctokit} from '@actions/github'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
import {getOctokit} from "@actions/github"
```